.github/workflows/labels.yml

```diff
@@ -0,0 +1,52 @@
+name: Add PR support labels
+
+on:
+  pull_request:
+    paths:
+      - "packs/**"
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  get-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+
+      - name: Cache dependencies
+        id: yarn-cache
+        uses: actions/cache@v2
+        with:
+          path: "**/node_modules"
+          key: ${{ runner.os }}-node-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile
+
+      - name: Check support level
+        id: support
+        run: |
+          URL="https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls/${{ github.event.pull_request.number }}/files"
+          yarn check-support-level $URL
```

Comment:
i think the yarn steps are gonna need cd utils, since we moved the javascript stuff into the utils folder.

Reply:
i also dropped this comment on ruairi's pr. if you feel like using this:
i think there is a default you can set for working directory.

something like:
```yml
jobs:
  x-job:
    defaults:
      working-directory: utils
```

so that becomes the default directory when commands execute, and you dont need cd utils for every step.
im not 100% thats correct, but it should be something like that.
also dont feel like you have to, just dropping this an option.